init_coupler.cfg

```diff
@@ -10,14 +10,15 @@ star_radius_modern          = 0.1192            # R_sun, radius of star (TODAY)
 star_temperature_modern     = 2566.0            # K, temperature of star (TODAY)
 star_luminosity_modern      = 0.000553          # L_sun, luminosity of star (TODAY) used for Baraffe model
 star_age_modern             = 7.6e9             # yr, age of star (TODAY)
-star_spectrum               = spec_trappist-1.txt   # Stellar spectrum file at 1 AU (TODAY)
+star_spectrum               = input/spec_trappist-1.txt   # Stellar spectrum file at 1 AU (TODAY)
 star_btrack                 = BHAC15-M0p090000.txt
 
 
 ##### Planet configuration
 
+
 mean_distance               = 0.700     # AU, star-planet distance
-gravity                     = 9.81      # m/s^2, planet surface gravity
```

Comment:
Where is gravity now defined/calculated?

Reply:
Gravity is now calculated from the mass and radius of the planet when the config file is read by `ReadInitFile()`. I switched over because AEOLUS now takes in planet mass when creating the `atmos` object. Maybe we could change it so that you can pass in either one, and it'll work out the other variable.